packages/pf3-component-mapper/src/form-fields/input-group-fields.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { InputGroup } from 'patternfly-react';
+
+export const InputAddonButtonGroup = ({ fields, formOptions, ...rest }) => (
+  <InputGroup.Button>
+    { formOptions.renderForm(fields, formOptions) }
+  </InputGroup.Button>
+);
+
+InputAddonButtonGroup.propTypes = {
+  formOptions: PropTypes.object.isRequired,
+  fields: PropTypes.array.isRequired,
+};
+
+export const InputAddonGroup = ({ fields, formOptions, ...rest }) => (
+  <InputGroup.Addon>
+    { formOptions.renderForm(fields, formOptions) }
+  </InputGroup.Addon>
+);
+
+InputAddonGroup.propTypes = {
+  formOptions: PropTypes.object.isRequired,
+  fields: PropTypes.array.isRequired,
+};
+
+export default [
```

Comment:
Don't export values like this. It does not follow any es modules rules. Use default or named exports.